landscape/client/monitor/snapservicesmonitor.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from landscape.client import snap_http
+from landscape.client.monitor.plugin import DataWatcher
+from landscape.client.snap_http import SnapdHttpException
+
+
+class SnapServicesMonitor(DataWatcher):
+
+    message_type = "snap-services"
+    message_key = "services"
+    persist_name = message_type
+    scope = "snaps"
+
+    def register(self, registry):
+        self.config = registry.config
+        super().register(registry)
```

Comment:
you may need to set `self.run_interval` for the monitor to be invoked periodically, but I guess until we have a server version that accepts these messages it's kind of moot.

Reply:
Added a 60 seconds `run_interval`.

Also, seems like the monitors inherit a default 5 seconds `run_interval` from `DataWatcher > MonitorPlugin > BrokerClientPlugin.run_interval`